configs/ACCIF405/config.h

```diff
@@ -60,7 +60,6 @@
 #define GYRO_1_CS_PIN        PA4
 #define GYRO_1_SPI_INSTANCE  SPI1
 #define GYRO_1_ALIGN CW0_DEG
-#define GYRO_1_ALIGN_YAW 1800
```

Comment:
This one is wrong - only remove cases where GYRO_1_ALIGN and GYRO_1_ALIGN_YAW do match. These mismatches must be handled individually (or left to trigger error)